palak/src/components/Label.jsx

```diff
@@ -0,0 +1,10 @@
+export const Label = ({ type, className }) => {
+  return (
+    <div className="mb-2">
+      <label className={`text-sm mb-2 ${className}`}>{type} </label>
+      <span className="ml-1 text-red-500">*</span>
```

Comment:
Try to make this required * as a reusable component as it can be used at multiple places.

Reply:
Address this issue.